kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties

```diff
@@ -706,7 +706,7 @@ netModemToolTipModemIndentifier=Enter a unique name for the modem.
 netModemToolTipModemInterfaceNumber=A unique number that identifies a modem interface. For example: <br><br>An Interface # of 0 would name the modem interface ppp0<br><br>An Interface # of 1 would name the modem interface ppp1
 netModemToolTipDialString=Instructions for how the modem should attempt to connect.<br><br>A typical dial string for the detected modem is: <b>{0}</b>
 netModemToolTipDialStringDefault=Instructions for how the modem should attempt to connect.<br><br>Typical dial strings are:<br>* EVDO Modems => <b>atd#777</b><br>* HSPA Modems => <b>atd*99***1#</b><br>For LTE Modems: <br>* LE910C1-EU => <b>ATD*99***4#</b><br>* LE910C1-AP (Telstra) => <b>ATD*99***1# or ATD*99***5#</b><br>* LE910-EU1 and LE910EU-V2 => <b>ATD*99***2# or ATD*99***1# (modem power cycle may be required for PDP context 1)</b><br>* LE910NA-V2 (AT&T) => <b>ATD*99***1#</b><br>* LE910NA-V2 (Verizon Wireless) => <b>ATD*99***3#</b><br>* Quectel-EG25 => <b>ATD*99***4#</b><br>* Other modems or mobile network operators => <b>please consult the documentation of the modem manufacturer and the mobile network operator</b><br><br>NOTE: The number before '#' defines the PDP context to be used for this connection.<br><br>Click the button to display/select existing PDP profiles. 
-netModemToolTipApn=Modem access point name.
+netModemToolTipApn=Modem access point name (Mandatory parameter for standard profiles and optional for generic ones).<br><br>In the case of generic profiles, if left empty this value will be picked up from the Mobile Broadband Provider the modem is registered to.<br><br>In order to avoid misconfiguration, it is strongly suggested to set it manually.
```

Comment:
Just popping up to my mind. Does it make sense to split and have a tooltip depending on the device profile?
So stating that is mandatory in older profiles
And having another message where it states as optional for generic ones